src/main.py

```diff
@@ -1,13 +1,16 @@
-from networking import NetworkingManager
+#from networking import NetworkingManager
 from render import RenderingManager
-from database import DatabaseManager
+#from database import DatabaseManager
+from tkinter import *
```

Comment:
Is there a reason you are importing tkinter to the main.py file?

Reply:
Thought I would need it to be using the mainloop() function, but it's being imported through RenderingManager, so I'll delete it.